src/Components/StepTwo/StepTwo.js

```diff
@@ -16,48 +126,119 @@ const StepTwo = ({nextStep, prevStep}) => {
             <div className="secondInputFields">
 
                 <div className="emailInput">
-                <p><span>*</span> What's your email address?</p>
-                <input type="email" name="" id="" placeholder="Email"/>
+                    <p><span>*</span> What's your email address?</p>
+                    <input 
+                    type="text" 
+                    name="email" 
+                    placeholder="Email"
+                    value={inputs.email} 
+                    onChange={handleChange}/>
+                   {errors.email && <div style={{color: "red"}}>{errors.email?.email}</div>}
                 </div>
 
                 <div className="homeAddressInput">
-                <div className="addressLine">
+                    <div className="addressLine">
                     <p><span>*</span> Address Line 1</p>
-                    <input type="text" placeholder="Address"/>
-                </div>
-                <div className="addressLine">
+                    <input 
+                    type="text"
+                    name={"addressLine1"} 
+                    placeholder="Address" 
+                    value={inputs.address.addressLine1}
+                    onChange={handleChange}/>
+                    </div>
+                    <div className="addressLine">
                     <p><span>*</span> Address Line 2</p>
-                    <input type="text" placeholder="Address"/>
+                    <input 
+                    type="text" 
+                    placeholder="Address"
+                    name={"addressLine2"} 
+                    value={inputs.address.addressLine2} 
+                    onChange={handleChange}/>
+                    </div>
                 </div>
-            </div>
 
-            <div className="homeCityInput">
-                <div className="city">
-                    <p><span>*</span> City/Town</p>
-                    <input type="text" name="" id="" placeholder="City"/>
-                </div>
-                <div className="state">
+                <div className="homeCityInput">
+                    <div className="state">
                     <p><span>*</span> State</p>
-                    <input type="text" name="" id="" placeholder="State"/>
-                </div>
-                <div className="zip">
+                    <input 
+                    type="text" 
+                    name={"state"}
+                    list={"state"} 
+                    value={inputs.address.state} 
+                    placeholder="State"
+                    onChange={handleChange}/>
+                    <datalist id={"state"}>
+                    {dataStates.map((state) => (
+                    <option value={state.name || state.code} 
+                    key={state.name + state.code} />
+                      ))}
```

Comment:
It's good practice to have some type of code linter (ESLint, Prettier, etc...), so the code gets formatted properly.

Reply:
I definitely have it and I used it with 1st pull request. Right now I do see it in VS code but cant apply it on my code. Colleagues tried to help too, but no one know what is going on.

Reply:
Please see this [article](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), and try re-installing ESLint, as it is mandatory to have formatted code (There is no real life project without it). You don't need to include the ESLint in the project setup, just add it to settings,json for Visual Studio Code.